### DISCUSSION TITLE
Making the Sidebar Collapsible in Orchid

### DISCUSSION THREAD

**Discussion by uzmanwebmaster** [orchidsoftware/platform#2893] [Q&A]
Hi,

I am developing an application with Laravel Orchid, and I would like to make the **sidebar collapsible**. Specifically, I want to implement a feature where the sidebar can be toggled (opened and closed) by clicking or through some other action.

Is there a built-in feature in Orchid to achieve this, or would you recommend a particular method or approach?

Any example code or guidance would be greatly appreciated.

Thank you in advance for your help!

**Comment by markc** [comment]
I'd like to suggest that aside from making the LHS sidebar collapsible, that it also has an option to "float" or be "pinned" to the layout. Also, if this is possible, then ALSO having a RHS sidebar with the same float/pinned options would provide a lot of flexible possibilities. As for toggling these sidebars, the most obvious would be to have a typical top horizontal bar (that could also be a branded navbar with links) with hamburger icons in the top left and top right corners to toggle the respective sidebars.

**Comment by iamB0rgy** [comment]
So this means, that this isn't responsive by default? shame!
